src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <UWPCompatible>true</UWPCompatible>
```

Comment:
You probably also want to set this for the other netstandard versions so the <2.0 netstandard build configurations get the same treatment.